ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -195,6 +224,34 @@ extension SignalProtocol {
 	}
 }
 
+extension SignalProtocol where Self.Value: OptionalProtocol {
+	/// Create a `RACSignal` that will observe the given signal.
+	///
+	/// - note: Any `interrupted` events will be silently discarded.
+	///
+	/// - returns: `RACSignal` instantiated from `self`.
+	public func toRACSignal() -> RACSignal {
```

Comment:
Could you delegate these overloads to the original implementations to avoid the code duplication?